packages/expo-updates/ios/Tests/LegacyUpdateSpec.swift

```diff
@@ -7,17 +7,17 @@ import ExpoModulesTestCore
 import EXManifests
 
 class LegacyUpdateSpec : ExpoSpec {
```

Comment:
Unrelated to this PR but do we need LegacyUpdate tests? Holding on to them till SDK 51's Expo Go drops 49?

Reply:
Yep.